arrays/arrays.c

```diff
@@ -21,22 +21,33 @@ typedef struct Array {
  *****/
 Array *create_array (int capacity) {
   // Allocate memory for the Array struct
+  Array *arr = malloc(sizeof(Array));
 
   // Set initial values for capacity and count
+  arr->capacity = capacity;
+  arr->count = 0;
 
   // Allocate memory for elements
+  arr->elements = malloc(capacity * sizeof(char *));
 
+  return arr;
 }
 
 
 /*****
  * Free memory for an array and all of its stored elements
  *****/
 void destroy_array(Array *arr) {
```

Comment:
Excellent, you were careful to free ALL of the alloc'd data here. No memory leaks! 💯